### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by Kamba56**
Setup webpack
Added html, css and js files
Added a module that will populate the markup dynamically

### CODE REVIEW

File: src/modules/populate.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  text.setAttribute('disabled', '');
+
+  const div = document.createElement('div');
+  div.append(checkbox, text);
+
+  const btn = document.createElement('button');
+  btn.setAttribute('type', 'button');
+  btn.innerHTML = '<i class="bi bi-three-dots-vertical"></i>';
+  listItem.append(div, btn);
+};
```
Comment (ShadyShawkat) [inline src/modules/populate.js:24]: - [ ] Kindly make sure that the list is populated in the DOM dynamically in the order of the index property, as per [this](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.) requirement,

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @Kamba56, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/modules/populate.js (hunk 1 above)
Comment (Kamba56) [reply src/modules/populate.js:24]: @ShadyShawkat  I don't understand what you mean because it is already populated in the order of the index

Comment (ShadyShawkat) [reply src/modules/populate.js:24]: Hey @Kamba56, the requirements says to populate the DOM in the order of the index values. that means if you were to have mixed values for the indices, you first would want to sort them according to their index values and then populate the DOM.

**Review by Ismailco — APPROVED** [review]
### Hi @Kamba56,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**